lib/matplotlib/tests/test_axes.py

```diff
@@ -2930,9 +2930,9 @@ def test_hist_stacked_normed():
     # make some data
     d1 = np.linspace(1, 3, 20)
     d2 = np.linspace(0, 10, 50)
-    fig = plt.figure()
-    ax = fig.add_subplot(111)
-    ax.hist((d1, d2), stacked=True, normed=True)
+    fig, ax = plt.subplots()
+    with pytest.warns(UserWarning):
+        ax.hist((d1, d2), stacked=True, normed=True)
```

Comment:
Instead of catching the warning, why not stop using the deprecated 'normed' kwarg in the tests?

Reply:
Because although it's deprecated, it should still work until removed (and we should be testing that it still works)